compute/Dockerfile.compute-node

```diff
@@ -445,10 +501,9 @@ FROM build-deps AS hll-pg-build
 ARG PG_VERSION
 COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
 
-RUN case "${PG_VERSION}" in "v17") \
-    echo "v17 extensions are not supported yet. Quit" && exit 0;; \
-    esac && \
-    wget https://github.com/citusdata/postgresql-hll/archive/refs/tags/v2.18.tar.gz -O hll.tar.gz && \
+# not version-specific
+# last release v2.18 - Sep 15, 2023
```

Comment:
```suggestion
# last release v2.18 - Aug 29, 2023
```
according to https://github.com/citusdata/postgresql-hll/blob/master/CHANGELOG.md. According to github, the tag was created Aug 31, 2023.